packages/maskbook/src/components/InjectedComponents/ToolboxHint.tsx

```diff
@@ -179,6 +180,12 @@ export function ToolboxHint(props: ToolboxHintProps) {
     const { onOpen: openSwapDialog } = useRemoteControlledDialogEvent(PluginTraderMessages.events.swapDialogUpdated)
     //#endregion
 
+    //#region Claim All ITO
+    const [isClaimAllDialogOpen, setClaimAllDialogOpen] = useState(false)
```

Comment:
Do you wanna try a hook named `useControlledDialog` in the codebase?

Reply:
yes, it's cute.